### ISSUE TITLE
Accessibility Checker Reports False SIA-R113 Errors for Hidden Elements

### ISSUE THREAD

**Issue by MaksimShakavin** [Siteimprove/alfa#1639]
DISCLAIMER: most likely issue is connected to a shadow dom and web-components handling. 

When scanning pages with the Siteimprove Accessibility Checker, false "Target Size (Minimum)" errors are reported for elements that are not visible on the page. These elements are reported to have a size of "0.00 x 0.00 pixels." A detailed inspection of these elements reveals that  their ancestors have the CSS property "display: none" set, which should exclude them from accessibility checks as they are not rendered.

### Steps to Reproduce:

- Navigate to the URL: https://sap.com/
- Use the Siteimprove Accessibility Checker to scan the webpage.
- Observe the reported 2.5.8 Target Size (Minimum) errors.
- Inspect the elements flagged with these errors using browser development tools.
- Confirm that one of their ancestor elements have "display: none" set.

### Expected Behavior:
Elements that are not rendered on the page (i.e., those with "display: none" or inside a parent element with "display: none") should not be evaluated for target size compliance under WCAG criteria since they are not visible or operable to users.

**Comment by Jym77** [comment]
Hi @MaksimShakavin this is due to constructed style sheets that we do not currently grab 🙈 
We have a fix ready in #1647 I hope to make a new release (v0.87.0) with it next week. The deployment in the extension will take a bit more time, likely later in July.

**Comment by rcj-siteimprove** [comment]
Hi @MaksimShakavin, we haven't forgotten this, we just still haven't been able to release it to the web extension yet. We'll notify you as soon as it has been released.

**Comment by Jym77** [comment]
Hi @MaksimShakavin, we've (finally) released the newer version of the extension (sorry, summer vacations delayed the work…)

I've just checked sap.com with it and no 2.5.8 errors were reported 🎉 
The new version has just been pushed to the extension stores, it may take a bit of time before it get updated on your machine.
We did encounter a problem with the Firefox extension, so that one is still on the old version, but the Chrome one has been updated.

I am closing this issue now, feel free to re-open if the problem is still here.
